src/hooks/useAuth.tsx

```diff
@@ -152,6 +158,11 @@ export const AuthContextProvider = ({
   }, [currentAppState, refreshIfNeeded]);
 
   const refreshForAuthFailure = useCallback(async () => {
+    // TODO: Add throttling, etc.
+    return refreshIfNeeded(true);
+  }, [refreshIfNeeded]);
+
+  const refreshForInviteAccept = useCallback(async () => {
```

Comment:
Just curious, what's your reasoning for having two identical "forceRefresh" methods?

Reply:
Check the comments above on line 20.  I believe the answer is so that it feels wrong to refresh for reasons other than these two.